modules/ckanext-ytp_main/ckanext/ytp/validators.py

```diff
@@ -55,6 +55,17 @@ def list_to_string(list):
     return list
 
 
+def string_to_list(value):
```

Comment:
The only difference with this (string_to_list) and the convert_to_list was that convert_to_list implicitly turns the strings to lowercase; which in this case of closed list with capital cased tag names breaks the javascript thingamajig that creates the tags on page load (since lowercased name doesn't match the initial capital cased name). 

After brief internal discussion the correct fix was determined to be the removal of the implicit lowercasing from the convert_to_list validator and do the lowercasing explicitly where it was needed.

Reply:
Fixed in dfe26c6c5169612da89367735aeeb03593839d65